sdk/src/message.rs

```diff
@@ -84,17 +76,19 @@ pub struct Message {
     /// signatures must match the first `num_required_signatures` of `account_keys`.
     pub num_required_signatures: u8,
 
+    /// The number of credit-only accounts: [signed_keys, unsigned_keys]. The last
+    /// `num_credit_only_accounts` of the signed (first `num_required_signatures`) and unsigned
+    /// portions of `account_keys` will be cached during runtime to allow parallel processing
+    /// across transactions.
+    pub num_credit_only_accounts: [u8; 2],
```

Comment:
I feel that splitting this array into two fields will be more readable and less error prone:
```rust
pub num_credit_only_signed_accounts: u8,
pub num_credit_only_unsigned accounts: u8,
```
Being more explicit like this avoids having to remember that index `0` is for signed, `1` for unsigned.   wdyt?

Reply:
I think that'll make the comment way more readable too (when split into two).  I had to read the current comment a couple times before I got it :)

Reply:
Fair enough, I'm convinced. Will push that change shortly!